### PR TITLE
qat examples with new api

### PR DESCRIPTION
**Pull request by n1ck-guo**
## Type of Change

Add and update qat examples with new api
Include:

* resnet_v2

## Description

JIRA: https://jira.devtools.intel.com/browse/ILITV-2209

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR: as titled 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed: None

### CODE REVIEW

File: examples/tensorflow/image_recognition/keras_models/densenet169/quantization/qat/main.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+        from neural_compressor.benchmark import fit
+        from neural_compressor.experimental import common
+        from neural_compressor.config import BenchmarkConfig
+        assert FLAGS.mode == 'performance' or FLAGS.mode == 'accuracy', \
+        "Benchmark only supports performance or accuracy mode."
+
+        model = common.Model(FLAGS.input_model).graph_def
+        if FLAGS.mode == 'performance':
+            conf = BenchmarkConfig(cores_per_instance=4, num_of_instance=7)
+            fit(model, conf, b_func=evaluate)
```
Comment (chensuyue) [inline examples/tensorflow/image_recognition/keras_models/densenet169/quantization/qat/main.py:177]: This example use b_func for benchmark, so please input '--iters' in 'run_benchmark.py' for performance mode.

File: examples/tensorflow/image_recognition/keras_models/mobilenet_v2/quantization/qat/main.py [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+        from neural_compressor.benchmark import fit
+        from neural_compressor.experimental import common
+        from neural_compressor.config import BenchmarkConfig
+        assert FLAGS.mode == 'performance' or FLAGS.mode == 'accuracy', \
+        "Benchmark only supports performance or accuracy mode."
+
+        model = common.Model(FLAGS.input_model).graph_def
+        if FLAGS.mode == 'performance':
+            conf = BenchmarkConfig(cores_per_instance=4, num_of_instance=7)
+            fit(model, conf, b_func=evaluate)
```
Comment (chensuyue) [inline examples/tensorflow/image_recognition/keras_models/mobilenet_v2/quantization/qat/main.py:177]: This example use b_func for benchmark, so please input '--iters' in 'run_benchmark.py' for performance mode.

File: examples/tensorflow/image_recognition/keras_models/resnet101/quantization/qat/main.py [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+        from neural_compressor.benchmark import fit
+        from neural_compressor.experimental import common
+        from neural_compressor.config import BenchmarkConfig
+        assert FLAGS.mode == 'performance' or FLAGS.mode == 'accuracy', \
+        "Benchmark only supports performance or accuracy mode."
+
+        model = common.Model(FLAGS.input_model).graph_def
+        if FLAGS.mode == 'performance':
+            conf = BenchmarkConfig(cores_per_instance=4, num_of_instance=7)
+            fit(model, conf, b_func=evaluate)
```
Comment (chensuyue) [inline examples/tensorflow/image_recognition/keras_models/resnet101/quantization/qat/main.py:177]: This example use b_func for benchmark, so please input '--iters' in 'run_benchmark.py' for performance mode.

File: examples/tensorflow/image_recognition/keras_models/resnetv2_101/quantization/qat/main.py [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+        from neural_compressor.benchmark import fit
+        from neural_compressor.experimental import common
+        from neural_compressor.config import BenchmarkConfig
+        assert FLAGS.mode == 'performance' or FLAGS.mode == 'accuracy', \
+        "Benchmark only supports performance or accuracy mode."
+
+        model = common.Model(FLAGS.input_model).graph_def
+        if FLAGS.mode == 'performance':
+            conf = BenchmarkConfig(cores_per_instance=4, num_of_instance=7)
+            fit(model, conf, b_func=evaluate)
```
Comment (chensuyue) [inline examples/tensorflow/image_recognition/keras_models/resnetv2_101/quantization/qat/main.py:177]: This example use b_func for benchmark, so please input '--iters' in 'run_benchmark.py' for performance mode.

File: examples/tensorflow/image_recognition/keras_models/resnetv2_50/quantization/qat/main.py [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+        from neural_compressor.benchmark import fit
+        from neural_compressor.experimental import common
+        from neural_compressor.config import BenchmarkConfig
+        assert FLAGS.mode == 'performance' or FLAGS.mode == 'accuracy', \
+        "Benchmark only supports performance or accuracy mode."
+
+        model = common.Model(FLAGS.input_model).graph_def
+        if FLAGS.mode == 'performance':
+            conf = BenchmarkConfig(cores_per_instance=4, num_of_instance=7)
+            fit(model, conf, b_func=evaluate)
```
Comment (chensuyue) [inline examples/tensorflow/image_recognition/keras_models/resnetv2_50/quantization/qat/main.py:177]: This example use b_func for benchmark, so please input '--iters' in 'run_benchmark.py' for performance mode.

**Comment by chensuyue** [comment]
[extension test](https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3866/)

**Comment by chensuyue** [comment]
[retest](https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3872/)

**Comment by chensuyue** [comment]
[retest after bug fix](https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3873/)

**Comment by chensuyue** [comment]
> [retest after bug fix](https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3875/)

Please check the invalid performance and accuracy.

**Comment by n1ck-guo** [comment]
retest

**Comment by n1ck-guo** [comment]
[retest](https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3950/)

**Comment by chensuyue** [comment]
[extension test](https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3984/artifact/report.html) , not ready.

**Comment by zehao-intel** [comment]
Rebuild extension for resnet50 and resnet101 after increasing the training epochs:https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3991/

**Comment by n1ck-guo** [comment]
skx-8180
batch_size: 100
resnet50: ori: 0.63272 qat: 0.61474
resnet101: ori: 0.63438 qat: 0.54072

**Comment by n1ck-guo** [comment]
[retest](https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4007/artifact/report.html)

**Review by zehao-intel — APPROVED** [review]

**Comment by chensuyue** [comment]
[retest](https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4410/)

**Comment by zehao-intel** [comment]
Extension test: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4415/
